src/commands/Information/userinfo.ts

```diff
@@ -122,114 +121,59 @@ export default class extends Command {
 				.setAuthor(`${user.username}#${user.discriminator}`, user.avatarUrl)
 				.setDescription(description)
 				.setFooter(
-					await this.client.bulbutils.translate("global_executed_by", message.guild?.id, {
-						user: message.author,
+					await this.client.bulbutils.translate("global_executed_by", context.guild?.id, {
+						user: context.author,
 					}),
-					<string>message.author.avatarURL({ dynamic: true }),
+					<string>context.author.avatarURL({ dynamic: true }),
 				)
 				.setTimestamp();
 
-			const msg = await message.channel.send({ embeds: [embed], components });
+			const msg = await context.channel.send({ embeds: [embed], components });
 
-			const filter = (i: any) => i.user.id === message.author.id;
+			const filter = (i: any) => !i.bot;
 			const collector = msg.createMessageComponentCollector({ filter, time: 15000 });
```

Comment:
15 seconds feels too short for these collectors to me.

Reply:
Can be changed to 30 seconds to match our other collectors